content/articles/creating-a-shopping-cart-api/index.md

```diff
@@ -0,0 +1,588 @@
+Building a dynamic web app, website, or mobile app requires the consumption of APIs (Application Programming Interfaces). Most companies provide APIs for carrying out some tasks such as building an eCommerce web app. For example, you can call/consume an API from the central bank of your country or some established applications like PayPal, Paystack, etc.
+<!--More -->
+In this tutorial, I will guide you through building a basic shopping cart API modeling how an actual shopping cart functions.
+In a real web app, you will be consuming APIs into this code that I have created, but for now, we have to use actual value as the amount in the shopping cart.
+You can always visit this link [Github](https://github.com/kingsleynwafor54/shopping_cart_with_springboot) to review the code base and check the disparity between yours and the tutorial.
```

Comment:
This doesn't fit into the intro all that well. I would remove this since you already mention this in the end.

Reply:
@kingsleynwafor54 I noticed you removed lines 3-4 here which would be this part:

> In this tutorial, I will guide you through building a basic shopping cart API modeling how an actual shopping cart functions.
> In a real web app, you will be consuming APIs into this code that I have created, but for now, we have to use actual value as the amount in the shopping cart.

I actually only meant for you to remove the fifth line which would be this part:

> You can always visit this link [Github](https://github.com/kingsleynwafor54/shopping_cart_with_springboot) to review the code base and check the disparity between yours and the tutorial.

I will be adding that part back for you. Sorry for the confusion.

Reply:
@kingsleynwafor54 Nevermind. I see what you did now. I'll only add back this one sentence:

> In this tutorial, I will guide you through building a basic shopping cart API modeling how an actual shopping cart functions.

instead.